src/theme.js

```diff
@@ -7,6 +7,6 @@
  * Then, in your `theme.config` file, change the following variable:
  * `@container   : 'pastanaga-cms-ui'`
  */
-// import '../theme/themes/pastanaga-cms-ui/extras/cms-ui.semantic.less';
-import 'semantic-ui-less/semantic.less';
+import '../theme/themes/pastanaga-cms-ui/extras/cms-ui.semantic.less';
+//import 'semantic-ui-less/semantic.less';
```

Comment:
I think this is some development leftover, this should be reverted because semantic.less is the default file that should be used

Reply:
Ah, noticed you fixed it while I was commenting :)